### ISSUE TITLE
[Enhancement] Custom log printing Topic

### ISSUE THREAD

**Issue by magestacks** [opengoofy/hippo4j#1230]
## Demand advice

Please answer these questions before submitting them. Thank you.

### Is your function request related to the problem?

No.

### Describe the function you want

The log is printed as follows.

```txt
WARN 26935 --- [           main] c.h.m.s.Hippo4jBaseSendMessageService    : [Tomcat] Please configure alarm notification on the server.
```

Hope to modify `c.h.m.s.Hippo4jBaseSendMessageService` to a more specific prompt, such as: `Hippo4j-Send-Message`.

```txt
WARN 26935 --- [           main] Hippo4j-Send-Message    : [Tomcat] Please configure alarm notification on the server.
```

TODO list:

- [ ] Sort out the classes that trigger log printing when Hippo4j runs.
- [ ] What is the name of the topic printed in the idea log.

**Comment by pc-mysql** [comment]
我想要认领这个任务

**Comment by pc-mysql** [comment]
I have a question on the issue, Should I change the just Hippo4jBaseSendMessageService log format one or others.

**Comment by pc-mysql** [comment]
@magestacks I have a question on the issue, Should I change the just Hippo4jBaseSendMessageService log format one or others.

**Comment by magestacks** [comment]
@pc-mysql Topic naming modification limited to `@Slf4j`.

**Comment by pc-mysql** [comment]
您的来信我已收到，将尽快给您回复，谢谢，pc

**Comment by pc-mysql** [comment]
@magestacks 原谅我还是有点不理解这个需求，是需要我列出所有在Hippo4j运行时会打印日志的所有类和topic吗，这大概会有50个以上的类？

**Comment by magestacks** [comment]
@pc-mysql Yes, there may be more classes. You can first discuss how to name the log topic

In terms of printing changelogs, is it going to be called `hippo4j-threadpool-dynamic` or `Hippo4j ThreadPool Dynamic`.

**Comment by DDreame** [comment]
please assign to me

**Comment by pirme** [comment]
@DDreame Welcome

**Comment by magestacks** [comment]
@DDreame Is there any progress?
